### PR TITLE
Issue 558: add discord integration

### PR DESCRIPTION
**Pull request by emanaton**
This update addresses issue 558's request for Discord integration. 

Note:
* the addition of `league/html-to-markdown` in composer in order to handle formatting messages in a way Discord can handle. 
* I didn't provide all the translations for the strings added to `language/en-US.ini` as I'm not sure how that's properly done; if that's something I can or should handle on my end, please let me know.

### CODE REVIEW

File: src/domain/projects/templates/showProject.tpl.php [hunk 1]
```diff
@@ -197,6 +197,28 @@
                                 </form>
                             </div>
                         </div>
+
+                        <?php // Slack webhook ?>
+                        <h4 class='widgettitle title-light'><span class='iconfa iconfa-leaf'></span>Discord</h4>
+                        <div class='row'>
+                            <div class='col-md-3'>
+                                <img src='<?= BASE_URL ?>/images/discord-logo.png' width='200'/>
```
Comment (marcelfolaron) [inline src/domain/projects/templates/showProject.tpl.php:205]: Can you add the discord logo to the images folder?

**Comment by marcelfolaron** [comment]
Thank you. This is great! I didn't see the discord logo. Can you add that to the folder? Other than that it looks good.

**Comment by emanaton** [comment]
Gah! Good catch, @marcelfolaron! Somehow my git-diff left that out. Thank you for catching that. Added.

**Comment by emanaton** [comment]
OH! And I fixed a small somewhat unrelated bug on line 65 of `class.showProject.php` that got included in this pull.

File: src/domain/projects/templates/showProject.tpl.php (hunk 1 above)
Comment (emanaton) [reply src/domain/projects/templates/showProject.tpl.php:205]: Done! Thank you for catching that.

**Comment by marcelfolaron** [comment]
Awesome, thanks!

**Review by marcelfolaron — APPROVED** [review]

**Pull request merged**